src/settings.js

```diff
@@ -1504,6 +1504,9 @@ var EMIT_EMSCRIPTEN_METADATA = 0;
 // library function in library_foo.js.
 var AUTO_JS_LIBRARIES = 1;
 
+// Internal: represents a browser version that is not supported at all.
+var TARGET_NOT_SUPPORTED = 0x7FFFFFFF;
```

Comment:
We have tried to avoid writing internal  settings in this file, after @sbc100 moved them to a new file, `settings_internal.js`.

I think we can move this to there, but may need to make sure to load the internal ones first in that case.

Reply:
Yeah, I saw the settings_internal.js, but that is indeed loaded after settings.js, and did not want to change the order. Is there a special reason for that order?

Reply:
argh, switching the order runs into issues as well. I'll drop the whole variable from here